test/lit/passes/optimize-instructions-gc.wast

```diff
@@ -612,8 +615,12 @@
     )
   )
 
+  ;; CHECK:      (func $nothing
+  ;; CHECK-NEXT:  (nop)
+  ;; CHECK-NEXT: )
   (func $nothing)
```

Comment:
this also changes behavior on a single-line function. lgtm, but was that intentional?

Reply:
Yes, I intentionally updated the regex to handle this correctly.